tests/models/mega/test_modeling_mega.py

```diff
@@ -313,6 +313,32 @@ def create_and_check_decoder_model_past_large_inputs(
         # test that outputs are equal for slice
         self.parent.assertTrue(torch.allclose(output_from_past_slice, output_from_no_past_slice, atol=1e-3))
 
+    def create_and_check_decoder_model_with_chunking(
+        self,
+        config,
+        input_ids,
+        token_type_ids,
+        input_mask,
+        sequence_labels,
+        token_labels,
+        choice_labels,
+        encoder_hidden_states,
+        encoder_attention_mask,
+    ):
+        config.use_chunking = True
+        config.output_attentions = True
+        config.attention_activation = "laplace"
+        config.chunk_size = input_ids.size(1) * 2
+
+        model = MegaForCausalLM(config).to(torch_device).eval()
+
+        result = model(
+            input_ids.repeat(1, 8),
+        )
```

Comment:
```suggestion
        result = model(input_ids.repeat(1, 8))
```
we coul include attention masks as well to make sure the logic holds for attention masks that are given as an input.

Reply:
In that case, will the passed-in attention mask have the size `[bs, seq_len]` or `[bs, chunk_size]`?